### DISCUSSION TITLE
AMELIORATIONS PAGE MESURE PHARE PROTEINES

### DISCUSSION THREAD

**Discussion by JenniferStephan** [betagouv/ma-cantine#1034] [Retours utilisateur·rice]
checklist par erwan

**Comment by ErwanGv** [comment]
Voir les proposition de modification en gras. Merci d'avance !

- [x]  Les sources de protéines doivent être diversifiées - entrée en vigueur : **1er janvier 2024 (pour les entreprises privées)** ou dans  "Pour qui?" : **Entreprise (1er janvier 2024)**

- [x] Menu végétarien - séparer en deux mesures : 

- [x]  **Menu végétarien hebdomadaire**, entrée en vigueur : **1er novembre 2019**, Pour qui ? : **Scolaire**

- [x]  **Option végétarienne quotidienne**, entrée en vigueur : **1er janvier 2023**, Pour qui ? **Etat, Établissements publics, Entreprises publiques nationales** _on attend encore l'avis juridique pour le périmètre "Etat"_ , Description : **"Les restaurants de l’Etat, ses établissements publics, et les entreprises publiques nationales proposent une option végétarienne quotidienne, dès lors qu'ils proposent habituellement un choix multiple de menus, à partir du 1er janvier 2023."**

- [x]  Expérimentation du menu végétarien dans les collectivités volontaires - Entrée en vigueur : **du 24 août 2022 au 24 août 2024** 

- [x]  Pour qui ? **Collectivité territoriales volontaires** 
- [x]  Description : Pour les collectivités territoriales volontaires, une expérimentation suite à la loi Climat propose d’expérimenter, une option végétarienne quotidienne pour une durée de 2 ans, **sous réserve des conditions fixées par voie réglementaire destinées à garantir l'équilibre nutritionnel des repas servis et du respect d'exigences adaptées aux besoins des usagers**

- [x]  Dans les ressources : Vous pouvez accéder à un guide **sur le** menu végétarien **hebdomadaire en restauration scolaire** ou notre livret de recettes pour menus végétariens dans nos ressources. 
- [x]  Télécharger le **cadre général** du plan **pluriannuel** de diversification des **sources de** protéines.

**Comment by ErwanGv** [comment]
Quelques nouvelles demandes, notamment pour corriger une erreur de ma part (je m'y perds dans les années...) :
- Menu végétarien hebdomadaire : supprimer "Les restaurants de l’Etat et ses établissements publics proposent une option végétarienne quotidienne à partir du 1er janvier 2023."
- Expérimentation du menu végétarien dans les collectivités volontaires - Entrée en vigueur : du 24 août **2021** au 24 août **2023**

Merci d'avance !

**Comment by JenniferStephan** [comment]
@ErwanGv c'est pris en compte et en ligne sur https://ma-cantine-staging.cleverapps.io/mesures-phares/diversification-des-menus

j'ai remis le label "résolu" mais on peut l'enlever si on veut "resusciter" un sujet :)

https://www.loom.com/share/d75b548bf6904364bab0060f13ad6363

**Reply by ErwanGv** [reply]
Merci beaucoup !
